service/udp.go

```diff
@@ -129,7 +129,7 @@ func (s *udpService) Serve(clientConn net.PacketConn) error {
 		func() (connError *onet.ConnectionError) {
 			defer func() {
 				if r := recover(); r != nil {
-					logger.Errorf("Panic in UDP loop: %v", r)
+					logger.Errorf("Panic in UDP loop: %v. Continuing.", r)
```

Comment:
*nit* same as the comment in tcp.go

```suggestion
					logger.Errorf("Panic in UDP loop: %v. Continue listening...", r)
```

Reply:
updated